components/ui/__tests__/__snapshots__/Rating-test.tsx.snap

```diff
@@ -25,3 +25,20 @@ exports[`renders undefined value 1`] = `
   –
 </span>
 `;
+
+exports[`renders with a custom aria-label 1`] = `
+<span
+  aria-label="custom"
+  className="rating rating-B"
```

Comment:
Are we ok to have classes mixing lowercase and uppercase characters ?

Reply:
Ah yeah we could lower case these, but I'm a bit afraid to touch it, it's been like that for ages and I've no idea if it could be used somewhere else...